hoot-core-test/src/test/cpp/hoot/core/io/OgrReaderTest.cpp

```diff
@@ -64,9 +64,8 @@ class OgrReaderTest : public HootTestFixture
     {
       OgrReader uut;
 
-      Progress progress("runBasicTest");
       OsmMapPtr map(new OsmMap());
-      uut.read(_inputPath + "jakarta_raya_coastline.shp", "", map, progress);
+      uut.read(_inputPath + "jakarta_raya_coastline.shp", "", map);
 
```

Comment:
```Progress``` was only used in the test b/c ```OgrReader::read``` required it, although we don't really care about progress in the tests, so didn't add it back into them after I refactored the method.